### DISCUSSION TITLE
Does Zooniverse use some kind of database to track projects?

### DISCUSSION THREAD

**Discussion by bnord** [lsst-epo/citizen-science-notebooks#62] [Q&A] [answered]
Does Zooniverse use some kind of database to track projects?

**Comment by eatyourgreens** [comment]
Hi, I'm not quite sure what you mean by 'track projects' but the Zooniverse API is built on top of a Postgres database, which holds all our projects.

**Reply by bnord** [reply]
thank you!

What kinds of data is held in that db?

**Reply by eatyourgreens** [reply, marked as answer]
Here's a link to the projects API, which includes the fields that are available on every project. 

https://panoptes.docs.apiary.io/#reference/projects/project

**Reply by eatyourgreens** [reply]
We also track retirement at the workflow and subject set level. If a project has multiple subject sets or workflows, we can tell you which sets and workflows are fully classified (remembering that a given set may be fully classified on one workflow but still in progress on another.)

**Reply by eatyourgreens** [reply]
Here's a link to our new API docs, which will eventually replace the Apiary version. 

https://zooniverse.github.io/panoptes/#projects
